pkg/ecdsa/tss/protocol_announce.go

```diff
@@ -55,27 +57,16 @@ func AnnounceProtocol(
 			case <-ctx.Done():
 				return
 			case msg := <-announceInChan:
-				// Since broadcast channel has an address filter, we can
-				// assume each message come from a valid group member.
```

Comment:
Maybe we can leave that comment as it clarifies why we don't check the sender of the message.